### PR TITLE
Move complexity to base test + change functions to apply standards be…

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by iamchrissmith — DISMISSED** [review]
lgtm

File: src/DssSpell.t.base.sol [hunk 1]
```diff
@@ -1146,41 +1086,41 @@ contract GoerliDssSpellTestBase is Config, Test {
         assertEq(clip.wards(address(end)), 1);
 
         // Check toll in/out
-        assertEq(psm.tin(), tin, concat("Incorrect-tin-", _ilk));
-        assertEq(psm.tout(), tout, concat("Incorrect-tout-", _ilk));
+        assertEq(psm.tin(), tin * WAD / 10000, _concat("Incorrect-tin-", _ilk));
```
Comment (megozu) [inline src/DssSpell.t.base.sol:1089]: maybe consider computing the tin/tout rate from bps at the beginning to avoid redundancy

Comment (gbalabasquer) [reply src/DssSpell.t.base.sol]: Done. @iamchrissmith can I get a new approval please?

File: src/DssSpell.t.sol [hunk 2]
```diff
@@ -132,45 +128,21 @@ contract DssSpellTest is GoerliDssSpellTestBase {
...
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done(), "TestError/spell-not-done");
-
-        checkSystemValues(afterSpell);
-
-        checkCollateralValues(afterSpell);
-    }
-
     function testRemoveChainlogValues() private { // make private to disable
```
Comment (megozu) [inline src/DssSpell.t.sol:143]: nit: I would move this test after the `testNewChainlogValues`, but maybe a reordering could be done later in a separate PR

Comment (gbalabasquer) [reply src/DssSpell.t.sol:143]: Yes let's please move on.

**Review by megozu — DISMISSED** [review]
Tests PASS

LGTM

**Review by iamchrissmith — APPROVED** [review]
new changes look good

**Review by megozu — DISMISSED** [review]

**Review by megozu — APPROVED** [review]

**Pull request merged**